src/mobot_client/payments/payments.py

```diff
@@ -152,15 +153,12 @@ def get_unspent_pmob(self) -> int:
             unspent_pmob = int(account_amount_response["unspent_pmob"])
             return unspent_pmob
 
-    def has_enough_funds_for_payment(self, payment_amount: int) -> bool:
+    def has_enough_funds_for_payment(self, payment_amount: Decimal) -> bool:
         """Return a bool to check if we can pay out the desired amount"""
         return self.get_unspent_pmob() >= (
-                payment_amount + int(self.get_minimum_fee_pmob())
+                mc.mob2pmob(payment_amount) + int(self.minimum_fee_pmob)
         )
 
-    def get_minimum_fee_pmob(self) -> int:
```

Comment:
This function is in use in air_drop_session.py, causing it to break.

Reply:
Whoops, thanks! I put it back in.